convert/changeset.go

```diff
@@ -0,0 +1,16 @@
+package convert
+
+// ChangeDetector defines funcs for getting a changeset from two
+// instances of a class. This interface has to be implemented by
+// all
+type ChangeDetector interface {
+	ChangeSet(older ChangeDetector) ([]Change, error)
```

Comment:
Please add a `typedef ChangeSet []Change` to introduce a type and have the term "changeset" better reflected in the code.

Reply:
Done. b7fa5c0.

Reply:
@michaelkleinhenz you haven't added the `type` definition for `ChangeSet` in b7fa5c0 but in 5589d6827176aa5b0e6c8bef485051cca883e9bc. Thank you.